Kickstarter-iOS/Views/CommentInputContainerView.swift

```diff
@@ -20,12 +19,12 @@ final class CommentInputContainerView: UIView {
     CommentInputTextView(frame: .zero)
   }()
 
-  let placeholderLabel: UILabel = { UILabel(frame: .zero) }()
```

Comment:
This was moved to be non-alphabetical if you wanted to update that but otherwise lgtm!

Reply:
I was just trying to move the lazy vars above the private constants, but yea I can move this back above the lazy var.